models/topics.models.js

```diff
@@ -0,0 +1,8 @@
+const db = require("../db/connection.js");
+const format = require("pg-format");
+
+exports.fetchTopics = () => {
+  return db.query(`SELECT * FROM topics;`).then((results) => {
+    return results;
```

Comment:
Ideally we should only send back rows from the model, this will be a helpful pattern to follow later on when we are needing to check the rows for error handling purposes. Then in the controller we just send back the response rather than having to destructure it.